src/virtualenv.py

```diff
@@ -1635,9 +1635,9 @@ def fix_lib64(lib_dir, symlink=True):
     if os.path.lexists(lib64_link):
         return
     if symlink:
-        os.symlink(lib_dir, lib64_link)
+        os.symlink("lib", lib64_link)
```

Comment:
Should this be ``lib64`` on some systems? Please add a changelog entry too.

Reply:
it has always been `lib` (also `lib_dir` is an absolute path to `lib`), as I understand it, virtualenv always populates `lib` and creates `lib64` as a symlink to `lib` on platforms that search in `lib64`